src/Libraries/CoreNodeModels/Input/FileSystem.cs

```diff
@@ -107,7 +107,7 @@ private Filename(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts
             ShouldDisplayPreviewCore = false;
         }
 
-        public Filename() : base("Filename")
+        public Filename() : base(Resources.FilePathOutputDescription)
```

Comment:
@martinstacey I think you also need to update the node description by updating the `FilenameNodeDescription` resource string. Currently it says `Allows you to select a file on the system to get its filename.`. It should be changed to read `...file path`.

Reply:
Addressed in  49b8f20, also resource reference was corrected to point to `FilePathDescription`